core/src/components/picker-column-internal/picker-column-internal.scss

```diff
@@ -86,7 +91,7 @@
 }
 
 @media (any-hover: hover) {
-  :host(:focus) {
+  :host(:focus:not(.picker-column-disabled)) {
```

Comment:
Focusable items can't be focused if they are disabled (for example, `<button>`). I understand this is happening because we set `tabindex={0}` on the host, but it might make more sense to do `tabindex={disabled ? null : 0}` in the template instead. It's not a huge deal, but the selector here is confusing given how regular interactive elements work.

Reply:
Great idea, thanks! This change helped me be able to refactor some other stuff that hopefully makes everything more clear.